pubspec.yaml

```diff
@@ -4,7 +4,7 @@ description: The setup-dart GitHub Action - download and setup the Dart SDK.
 publish_to: none
 
 environment:
-  sdk: ^3.2.0-0
+  sdk: ^3.3.0-0
```

Comment:
This release doesn't exist yet. We shouldn't depend on it?

What's up with the -0 here actually?

Reply:
It's a semver idiom for the first `3.3.0` prerelease (or higher because of the caret). `0` sorts before any other digit or letter in ASCII sort order and is therefore the first allowed character in a prerelease identifier.